src/Features/Core/Portable/ConvertLinq/ConvertForEachToLinqQuery/ExtendedSyntaxNode.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.ConvertLinq.ConvertForEachToLinqQuery
+{
+    internal struct ExtendedSyntaxNode
+    {
+        public SyntaxNode Node { get; }
+
+        public ImmutableArray<SyntaxTrivia> ExtraLeadingComments { get; }
+
+        public ImmutableArray<SyntaxTrivia> ExtraTrailingComments { get; }
+
+        public ExtendedSyntaxNode(SyntaxNode node, IEnumerable<SyntaxToken> extraLeadingTokens, IEnumerable<SyntaxToken> extraTrailingTokens)
+         : this(node, Helpers.GetTrivia(extraLeadingTokens), Helpers.GetTrivia(extraTrailingTokens))
+        { }
```

Comment:
wrap.

Reply:
?

---
In reply to: [195846931](https://github.com/dotnet/roslyn/pull/27130#discussion_r195846931) [](ancestors = 195846931)

Reply:
In IDE code we basically never put ```{ }``` on the same line.  We always wrap them so that the ```}``` is on hte next line.  Thanks!

Reply:
OK. Will do. Thank you!

---
In reply to: [196607291](https://github.com/dotnet/roslyn/pull/27130#discussion_r196607291) [](ancestors = 196607291)